tests/protocol/create_order.ts

```diff
@@ -210,12 +215,12 @@ describe("Protocol - Create Order", () => {
       stakeInteger,
     );
 
-    const thrownError = orderResponse.errors[0] as AnchorError;
```

Comment:
I think losing the ability to surface the error codes is a bit rubbish, i.e., not being able to get at `thrownError.error.errorCode.code` any more. We should probably look at how to best surface them going forward if we're supporting sending our instructions as transactions and so any error back is a `SendTransactionError`.